test/E2ETest/Microsoft.Test.E2E.AspNet.OData/ServerSidePaging/ServerSidePagingControllers.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNet.OData;
+using Microsoft.Test.E2E.AspNet.OData.Common.Controllers;
+
+namespace Microsoft.Test.E2E.AspNet.OData.ServerSidePaging
+{
+    public class ServerSidePagingCustomersController : TestODataController
+    {
+        private readonly IList<ServerSidePagingCustomer> _serverSidePagingCustomers;
+
+        public ServerSidePagingCustomersController()
+        {
+            _serverSidePagingCustomers = new List<ServerSidePagingCustomer>(
```

Comment:
Nit: Enumerable.Range(1, 7).Select(i=> new SSPC{ })**.ToList()** :-)

Reply:
@g2mula I used `.ToList()` at first, then changed to call the constructor instead - my love for `.ToSomething()` is not that much for no good reason.